requirements.txt

```
@@ -1,7 +1,7 @@
 pip
-docutils==0.16
-sphinx==4.3.2
-sphinx-tabs
+docutils==0.18.0
```

lol https://github.com/sphinx-doc/sphinx/issues/10596

```suggestion
docutils==0.18.1
```